plugin/adsk/plugin/CMakeLists.txt

```diff
@@ -93,6 +87,16 @@ if(IS_MACOSX OR IS_LINUX)
     mayaUsd_install_rpath(rpath ${TARGET_NAME})
 endif()
 
+# -----------------------------------------------------------------------------
+# promoted headers
+# -----------------------------------------------------------------------------
+mayaUsd_promoteHeaderList( 
+    HEADERS
+        ${headers}
+    BASEDIR
+        ${TARGET_NAME}
+)
+
```

Comment:
I don't think these need to be promoted, or should, be promoted - see my comments overall review comments.